testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/preload/PreloadMiniAppWindow.kt

```diff
@@ -146,6 +186,7 @@ class PreloadMiniAppWindow(
     private companion object {
         const val LABEL_VERSION = "Version: "
         const val LABEL_CUSTOM_METADATA = "Custom MetaData: "
+        const val LABEL_RAE_SCOPES = "RAE Access Token Scopes: "
```

Comment:
Please remove the `:` and change the style of this element so it can be displayed as a title like the permissions title style.

```suggestion
        const val LABEL_RAE_SCOPES = "RAE Access Token Scopes"
```

Reply:
Thank you for the suggestion. After another discussion in channel, I redesigned this feature. Please see the updated description in PR for the screenshot.